src/etc/inc/dyndns.class

```diff
@@ -702,6 +703,19 @@
 					curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
 					curl_setopt($ch, CURLOPT_URL, $server . '?domains=' . str_replace('.duckdns.org', '', $this->_dnsHost) . '&token=' . $this->_dnsUser);
 					break;
+				case 'googledomains':
+					if ($this->_dnsVerboseLog)
+						log_error("Google Domains: ({$this->_dnsHost}): DNS update() starting.");
+					$server = "https://domains.google.com/nic/update";
+					$post_data['hostname'] = $this->_dnsHost;
+					$post_data['myip'] = $this->_dnsIP;
+					$post_data['offline'] = 'no';
+					curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, FALSE);
```

Comment:
This is probably the best option for now - with an entry in the release notes that it will work only on OpenSSL until the LibreSSL patch is released due to a bug in LibreSSL.
With `CURLOPT_SSL_VERIFYPEER` set to false it is insecure and may be attacked by a man in the middle attack and therefore the token could be stolen. In my opinion insecure code cannot be merged.

@fichtner  What do you think?